metrics/grep.csv

```diff
@@ -0,0 +1,3 @@
+Name,Labels,Description,Child,Initialize
+wmo_wis2_grep_connected_flag,centre_id|report_by,Connection status from GRep to to centre,-,false
+wmo_wis2_grep_api_status,centre_id,Status of GRep API,-,false
```

Comment:
Who would feed this metrics ? If the GR is dead ? It cannot say "I'm dead" :)
In fact, we are touching upon having a SCGR to check that the GR is working. No ?

Reply:
In theory, a global service can emit metrics while one of its other components are down?